### ISSUE TITLE
Any plans to update this library?

### ISSUE THREAD

**Issue by xleon** [rgonek/Ilaro.Admin#30]
I use django in some projects instead of .NET just to use the admin. This project looks great. Is it dead? any plans to continue?

**Comment by Rodbourn** [comment]
I wouldnt make a stack decision based on this library.  I've used it before, but it required some pretty hefty updates.

**Comment by xleon** [comment]
Of course not. I just interested to know if there are any future plans.

**Comment by Rodbourn** [comment]
It's been pretty stagnant for two'sh years - I wouldnt expect them :)

**Comment by rgonek** [comment]
I'll definitely still developing it, but unfortunately, a day have only 24 hours :), and lately after work, I've been doing some commercial projects, so I cannot promise anything besides that I'll do something with it.

From good news from the perspective of functionalities I'm a quite close to something that I could call version 1.0, but unfortunately, there is also a lot of bugs.

**Comment by Rodbourn** [comment]
It's a lovely project.  I would love to see it production ready.  In a pinch, it is good for very quickly giving an admin CRUD access to a large DB.

**Comment by xleon** [comment]
It is. I love the idea of Django admin applied to .NET. I seriously don´t understand why nobody else is doing stuff similar to this library. I thought about doing it myself, but as you know, this involves huge amount of time. I would definitely contribute if the project continues. 

One question:
Why did you choose `EntityConfiguration<T>` instead of `[Attribute()]` in the very same model? => That would be more in sync with django

**Comment by rgonek** [comment]
You can still use attribute configuration, and you can also use this kind of notation:
`Entity<OrderDetail>.Register().Table("Order Details");`

Not using attributes for configuration has one big advantage. If you have an existing project your rather wouldn't like to add attributes to your entities, or maybe even you can do that.

Having a few ways to do the same thing makes confusion, so it'll be probably a good idea to make it pluggable.
For example, to be able to use attribute configuration you would have to include IlaroAdmin.AttributesConfiguration.dll

**Comment by xleon** [comment]
Yeah, that makes sense from a .NET perspective. POCOs could not be used for anything else.
